osf/models/analytics.py

```diff
@@ -136,9 +136,27 @@ def update_counter(cls, page, node_info):
             model_instance.save()
 
     @classmethod
-    def get_basic_counters(cls, page):
+    def _get_file_counter(cls, counter_type, file_id, version=None):
+        """
+        Pulls the total file count for views or downloads across different nodes.
+        :param info_type: String
+        :param file_id: String
+        :param version: Int the version number
+        :return: Int the total count
+        """
+        if version is not None:
+            file_id += ':' + str(version)
+
         try:
-            counter = cls.objects.get(_id=cls.clean_page(page))
-            return (counter.unique, counter.total)
+            return PageCounter.objects.filter(_id__startswith=counter_type,
```

Comment:
Why is it necessary to aggregate the total? `PageCounter` `_id`'s are unique so wouldn't this be something like: 
```
counter = PageCounter.objects.get(_id__startswith=counter_type, _id__endswith=file_id)
return (counter.unique, counter.total)
```

Reply:
Oh wait... nevermind. There might be several records with a certain file id in the `_id` but with different node ids.